modulus/distributed/manager.py

```diff
@@ -26,6 +26,32 @@
 from modulus.distributed.config import ProcessGroupConfig, ProcessGroupNode
 
 
+class ModulusUndefinedGroupError(Exception):
+    """Exception for querying an undefined process group using the Modulus DistributedManager"""
+
+    def __init__(self, name: str):
+        """
+
+        Parameters
+        ----------
+        name : str
+            Name of the process group being queried.
+
+        """
+        message = (
+            f"Cannot query process group '{name}' before it is explicitly created."
+        )
+        super().__init__(message)
+
+
+class ModulusUninitializedDistributedManagerWarning(Warning):
+    """Warning to indicate usage of an uninitialized DistributedManager"""
+
+    def __init__(self):
+        message = "Instantiating DistributedManager before calling DistributedManager.initialize is not recommended"
+        super().__init__(message)
+
+
 class DistributedManager(object):
     """Distributed Manager for setting up distributed training enviroment.
```

Comment:
"enviroment" -> "environment" here and in other places.

Reply:
Fixed